src/scripts/genEtwProvider.py

```diff
@@ -64,7 +64,7 @@ def genProviderInterface(manifest, intermediate):
     for pattern, replacement in replacements:
         header_text = re.sub(pattern, replacement, header_text)
 
-    with open_for_update(path.join(provider_dirname, mcheader_filename)) as mcheader_file:
+    with open(path.join(provider_dirname, mcheader_filename), 'w') as mcheader_file:
```

Comment:
`open_for_update` is a function I wrote to only write to disk if the file would change. It is not relevant here because the file on disk will always end up being different. Both are functionally equivalent. I am changing it for clarity